src/components/ExpensifyCashLogo.js

```diff
@@ -3,26 +3,41 @@ import PropTypes from 'prop-types';
 import ProductionLogo from '../../assets/images/expensify-cash.svg';
 import DevLogo from '../../assets/images/expensify-cash-dev.svg';
 import StagingLogo from '../../assets/images/expensify-cash-stg.svg';
-import CONFIG from '../CONFIG';
 import CONST from '../CONST';
+import Environment from '../libs/Environment/index';
```

Comment:
```suggestion
import Environment from '../libs/Environment';
```
It will automatically find `index`.